ui/partials/perf/graphsctrl.html

```diff
@@ -32,6 +32,12 @@
     <div id="data-display">
       <select ng-model="myTimerange" ng-options="timerange.text for timerange in timeranges track by timerange.value" ng-change="timeRangeChanged()" ng-disabled="seriesList.length == 0">
       </select>
+      <select ng-model="myMeasure" ng-change="myMeasureChanged()">
```

Comment:
Could we disable this if seriesList.length == 0, as we do for timerange above? You should just need to add this to the tag:

    ng-disabled="seriesList.length == 0"